Core/forms.py

```diff
@@ -165,3 +169,27 @@ def __init__(self, *args, **kwargs):
                 'type': 'checkbox'}
             )
         )
+
+    def clean(self):
+        registered = None
+        email_exist = None
+        student_number = self.cleaned_data['student_number']
+        email = self.cleaned_data['email']
+        student_number = int(student_number)
+        event = Event.objects.filter(active=True).first()
+        try:
+            registered = Registration.objects.filter(student_number=student_number, event=event)
+        except:
```

Comment:
The above query will never raise any exception because of two reasons:
1. `.filter` returns a queryset, so even if it is empty, no exception would be raised
2. `.filter` returns a queryset which is lazy, so it wouldn't have even evaluated until this point

Solution:
This validation would not be needed after adding a call to `super` because the model level validation of unique together would be checked in `super` itself. So remove this altogether.